### PR TITLE
refactor: use @artus-cli/artus-cli

### PR DESCRIPTION
**Pull request by fengmk2**
closes https://github.com/eggjs/egg/issues/5140

Get Started https://www.yuque.com/atian25/blog/artus-cli

BREAKING CHANGE: drop Node.js 14 support and more deprecated features

### CODE REVIEW

**Comment by fengmk2** [comment]
--inspect-brk 好久没玩

image

File: bin/cli.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+
+import path from 'node:path';
+import { start } from '@artus-cli/artus-cli';
+
+start({ baseDir: path.dirname(__dirname) });
```
Comment (atian25) [inline bin/cli.ts:6]: 入参可以省略，会自己寻址

File: lib/cmd/test.js [hunk 2]
```diff
@@ -1,212 +0,0 @@
...
-      context.env.ENABLE_MOCHA_PARALLEL = 'true';
-    }
-    if (context.argv.parallel && context.argv['auto-agent']) {
-      context.env.AUTO_AGENT = 'true';
-    }
-
-    debug('run test: %s %s', mochaFile, testArgs.join(' '));
-    const opt = {
-      env: Object.assign({
-        NODE_ENV: 'test',
```
Comment (fengmk2) [inline lib/cmd/test.js:91]: 这个逻辑遗漏了。

**Comment by fengmk2** [comment]
https://github.com/cnpm/cnpmcore/pull/403 验证

**Pull request merged**